cobbler/actions/mkloaders.py

```diff
@@ -116,9 +133,23 @@ def mkimage(
     cmd = ["grub2-mkimage"]
     cmd.extend(("--format", image_format))
     cmd.extend(("--output", str(image_filename)))
+    cmd.append("--prefix=")
     cmd.extend(modules)
 
-    # The Exception raised by subprocess already contains everything useful,
-    # it's simpler to use that than roll our own custom exception together
-    # with cobbler.utils.subprocess_* functions
+    # The Exception raised by subprocess already contains everything useful, it's simpler to use that than roll our
```

Comment:
What line length are we aiming at?

Reply:
We defined in the prospector config that we aim for 120 chars

Reply:
I didn't even know what prospector is let alone that it's used :see_no_evil: Personally, I think that 120 is a bit long for having two panes next to each other, at least on my screens. I'd put 100 as the limit if this was my project.

BTW, what do you think about black? takes care of basically all formatting, the only configurable aspect is the line length. It's getting more and more popular in Python and basically brings the `gofmt` experience to Python (no one loves `gofmt`'s formatting but everyone loves that `gofmt` exists and can't be configured)

Reply:
I have nothing against introducing it. But let's not do it in this PR please.